tests/storage/datavolume.go

```diff
@@ -884,6 +884,19 @@ var _ = SIGDescribe("[Serial]DataVolume Integration", func() {
 	})
 
 	Context("Fedora VMI tests", func() {
+		imageSizeEqual := func(a, b int64) bool {
+			// Our OCS image size method probe is very precise and can show a few
+			// bytes of difference.
+			// A VM cannot do sub-512 byte accesses anyway, so such small size
+			// differences are practically equal.
+			if a-b < 512 {
```

Comment:
I am not sure this is always correct 
0 - 1000 = -1000

maybe  -> https://golang.org/pkg/math/#Abs

Reply:
haha, oops